src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  align-items: center;
+  width: 96px;
+  height: 16px;
+
+  &__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
```

Comment:
Replace with -n + 1 to make code the same style.

Reply:
I have a problem with moving to * selector. When i do it my code is crashed. Can I left it like now and in next task I`ll to do it from start

Reply:
@myehor what do you mean by _crash_?
what's wrong with just adding
```
* {
  box-sizing: border-box;
}
```